communities/src/migrations.rs

```diff
@@ -181,7 +325,8 @@ pub mod v1 {
 #[cfg(feature = "try-runtime")]
 mod test {
```

Comment:
tests now only cover the scenarion 0 -> 2, would be good to also have 1-> 2

Reply:
agreed

Reply:
test added